### PR TITLE
Add service call to publish vdbvolume as sensor_msgs::PointCloud2

### PR DESCRIPTION
**Pull request by rohitmenon86**
I have added a Service call to enable the publishing of the extracted vertices as ROS sensor_msgs::PointCloud2 on a topic for publishing fused pointclouds.

### CODE REVIEW

**Comment by nachovizzo** [comment]
Thanks @rohitmenon86 ! LGTM
Waiting on Gupta's approval now

**Review by nachovizzo — APPROVED** [review]

**Comment by saurabh1002** [comment]
@rohitmenon86 any updates on this PR? I can make the suggested changes and merge this if you give access to your fork.

**Comment by saurabh1002** [comment]
Closing due to inactivity.

**Pull request closed without merging**
